src/main/java/com/softserve/actent/controller/EquipmentController.java

```diff
@@ -25,72 +24,64 @@
 
     private final UserRepository userRepository;
 
-    private final EventRepository eventRepository;
+    private final EventServiceImpl eventService;
 
     private final ModelMapper modelMapper;
 
     @Autowired
-    public EquipmentController(EquipmentServiceImpl equipmentServiceImpl, UserRepository userRepository, EventRepository eventRepository, ModelMapper modelMapper) {
+    public EquipmentController(EquipmentServiceImpl equipmentServiceImpl, UserRepository userRepository, EventServiceImpl eventService, ModelMapper modelMapper) {
         this.equipmentServiceImpl = equipmentServiceImpl;
         this.userRepository = userRepository;
-        this.eventRepository = eventRepository;
+        this.eventService = eventService;
         this.modelMapper = modelMapper;
     }
 
     @GetMapping("/equipments")
-    public ResponseEntity<List<EquipmentDto>> getAllEquipments(){
+    @ResponseStatus(HttpStatus.OK)
+    public List<EquipmentDto> getAllEquipments(){
 
         List<Equipment> equipments = equipmentServiceImpl.getAll();
 
-        List<EquipmentDto> equipmentDtos = equipments.stream()
+        return equipments.stream()
```

Comment:
Mapping list of objects in controller might be "hard" operation

Reply:
We decided to map a dto to entity and vice versa only in controllers